public/css/flags/flags.css

```diff
@@ -93,10 +93,14 @@
     background-position: -80px -11px;
 }
 
-.flag.flag-multiple {
+.flag.flag-id {
     background-position: -80px -22px;
 }
 
+.flag.flag-multiple {
+    background-position: -80px -33px;
```

Comment:
Set two css variable for the height and width of a flag. Then you can also use them to determine which flag to show. :p